test/fail_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+NC='\033[0m'
+CYAN='\033[0;36m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+
+INPUT_FILES="compiler/fail/*.ods"
+TMP_FILE="temp.py"
```

Comment:
we should use `mktemp` here (see the compiler test file) to prevent naming conflicts. TODO: add flag to compiler so we don't need to compile to a file